src/internal/packager/deploy.go

```diff
@@ -93,6 +96,22 @@ func Deploy() {
 		os.Exit(0)
 	}
 
+	// @JPERRY set up installed package secret state info
```

Comment:
Check if this package has already been deployed.

We might need to add some more logic to know what to do when the package we're trying to install is already installed into the cluster.